doc/source/_templates/layout.html

```diff
@@ -0,0 +1,48 @@
+<!-- https://github.com/tholman/github-corners -->
+<!-- https://stackoverflow.com/a/53329712/10241324 -->
+{% extends '!layout.html' %}
+{% block document %}
+{{super()}}
```

Comment:
```suggestion
{{ super() }}
```
I meant like this. An HTML formatter probably won't do anything because the `{{` aren't special in HTML -- they are part of the templating language, not of HTML.